package.json

```diff
@@ -62,7 +62,13 @@
     "i18n:get-translations": "node src/locales/scripts/get-translations",
     "i18n:update-locales": "node src/locales/scripts/get-validated-locales",
     "i18n:generate-pot": "node src/locales/scripts/json-to-pot",
-    "i18n:check": "vue-i18n-extract report --vueFiles './src/*/**.?(js|ts|vue)' --languageFiles './src/locales/en.json'"
+    "i18n:check": "vue-i18n-extract report --vueFiles './src/*/**.?(js|ts|vue)' --languageFiles './src/locales/en.json'",
+    "create-component-sfc": "remake component",
```

Comment:
I wonder if we could follow the example of other scripts and add a prefix to group similar commands. `create:`, maybe? So it would be `create:story`, `create:component-storybook-test`, `create:component-unit-test` etc...

Reply:
yeah, sure